src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationView.cs

```diff
@@ -5017,6 +5087,53 @@ private bool IsFullScreenOrTabletMode()
 			return isFullScreenMode || isTabletMode;
 		}
 
+		private void SetDropShadow()
+		{
+			var displayMode = DisplayMode;
+
+			if (displayMode == NavigationViewDisplayMode.Compact || displayMode == NavigationViewDisplayMode.Minimal)
+			{
+				if (m_shadowCaster is { } shadowCaster)
+				{
+					//TODO MZ: Check if shadow is supported on this target
```

Comment:
Phone call for MZ, is there an MZ in the building...

;)

Reply:
![NoSnowstormGIF](https://user-images.githubusercontent.com/1075116/125524308-9e45ec59-4c16-44c2-9eca-cd55b3039be6.gif)